server/planning/events/module.py

```diff
@@ -46,3 +47,16 @@
     ),
     rest_endpoints=RestEndpointConfig(resource_methods=["GET"], item_methods=["GET"]),
 )
+
+events_autosave_resource_config: ResourceConfig = ResourceConfig(
+    name="event_autosave",
+    data_class=EventResourceModel,
+    service=EventsAutosaveAsyncService,
+    mongo=MongoResourceConfig(
+        indexes=[
+            MongoIndexOptions(name="event_autosave_user", keys=[("lock_user", 1)], background=True),
```

Comment:
Will also need to explicitly set `unique=False` for these 2, and the 2 for Planning autosave